src/main/java/com/github/dockerjava/api/model/PropagationMode.java

```diff
@@ -0,0 +1,49 @@
+package com.github.dockerjava.api.model;
+
+/**
+ * The propagation mode of a file system or file: <code>shared</code>, <code>slave</code> or <code>private</code>.
+ *
+ * @see https://github.com/docker/docker/pull/17034
+ * @since 1.22
+ */
+public enum PropagationMode {
+    /** default */
+    mount_default(""),
```

Comment:
Problem is that the values are default and private, which is not allowed. So how should they be called then?
Do you have an example for the JsonProperty stuff? I tried to make this analogue to SELContext.java.